eclipse-sprotty/sprotty-website
# Jiden/langium interactive example

an interactive example for sprotty for the Statemachine language. 
This is using the monaco editor with a Langium language server which is implementing a sprotyy diagram server through the sprotty-langium integration. 
communication is realized through a custom modelSource using the monacoLanguageClient 

**How to test**
1. npm install
2. npm run build --workspaces
3. npm start
4. example should be available under [http://localhost:8080/showcase](http://localhost:8080/showcase)
